pt-br/tutorials/tour/pattern-matching.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: tutorial
+title: Correspondência de Padrões
+
+disqus: true
+
+tutorial: scala-tour
+num: 11
+
+tutorial-next: singleton-objects
+tutorial-previous: case-classes
+language: pt-br
+---
+
+_Nota de tradução: A palavra cujo o significado melhor corresponde a palavra `match` em inglês seria `correspondência`. Também podemos entender que `match` é como "coincidir" ou "concordar" com algo._
+
+Scala possui mecanismo de correspondência de padrão embutido. Isso permite realizar o match de qualquer tipo de dados any sort of data com a política de primeiro match. 
```

Comment:
Yup, it should be: `Isso permite realizar o match de qualquer tipo de dados com a política de primeiro match`.